server/data/data-api/src/main/java/org/apache/james/rrt/lib/Mapping.java

```diff
@@ -182,11 +180,9 @@ public String withoutPrefix(String input) {
         }
 
         public static boolean hasPrefix(String mapping) {
-            return mapping.startsWith(Regex.asPrefix())
-                || mapping.startsWith(Domain.asPrefix())
-                || mapping.startsWith(Error.asPrefix())
-                || mapping.startsWith(Forward.asPrefix())
-                || mapping.startsWith(Group.asPrefix());
+            return Arrays.stream(Type.values())
+                .filter(type -> !type.asPrefix().isEmpty())
```

Comment:
Can we hide that rather technical (ducplicated) line behind a `Type::hasPrefix` mehod?